src/ui/public/filter_bar/lib/map_script.js

```diff
@@ -14,8 +14,7 @@ define(function () {
           if (filter.meta.formattedValue) {
             value = filter.meta.formattedValue;
           } else {
-            value = filter.script.script.params.value;
-            value = field.format.convert(value);
+            value = '' + filter.script.script.params.value;
```

Comment:
Unfortunately removing the call to `convert` has the side effect of preventing field formatters from taking effect. For instance, in the screenshot below both `bytes` and `Double Bytes` are configured with a bytes field formatter, but only the non-scripted bytes field is display correctly.

![screen shot 2016-10-13 at 3 43 28 pm](https://cloud.githubusercontent.com/assets/6239176/19364631/edfa1e00-915c-11e6-8e97-62b010dcb8c9.png)

I think we need to detect when the script filter is a range, and do what map_range.js is doing in those cases https://github.com/Bargs/kibana/blob/cd760781b202e74c0dc83edcd8e36341043a42f7/src/ui/public/filter_bar/lib/map_range.js#L22-L22